docs/topics/installing-vs-code-plugin.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension for Visual Studio Code
+
+You can install the {ProductShortName} extension for Visual Studio Code (VS Code) on any platform that supports VS Code.
+
+.Prerequisites
+
+* 8 GB RAM
+* JDK 11 or later
+
+.Procedure
+
+. Set the environmental variable `JAVA_HOME` to `jdk11`:
++
+[source, terminal]
+----
+$ export JAVA_HOME=jdk11
+----
+
+. In VS Code, click the *Extensions* icon on the Activity bar to open the *Extensions* view.
+. Enter `{ProductName}` in the Search field.
+. Select the *Migration Toolkit for Applications* extension and click *Install*.
++
+The {ProductShortName} extension icon (image:vs_MTA_extension_icon.png[MTA code extension]) is displayed on the Activity bar.
+
+[NOTE]
+====
```

Comment:
Honestly, I am not sure we need this note. The MTA extension is uninstalled just like any other VSC extension. That procedure is documented in the VSC documentation.